source/common/upstream/cds_subscription.h

```diff
@@ -46,12 +46,15 @@ class CdsSubscription : public Http::RestApiFetcher,
     NOT_IMPLEMENTED;
   }
 
+  const std::string& versionInfo() override { return version_info_; }
+
   // Http::RestApiFetcher
   void createRequest(Http::Message& request) override;
   void parseResponse(const Http::Message& response) override;
   void onFetchComplete() override;
   void onFetchFailure(const EnvoyException* e) override;
 
+  std::string version_info_;
```

Comment:
To @mattklein123's testing point, it seems the legacy xDS adapters other than RDS don't have `version_info_` update code.